src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int AGE_OK = 19;
+    private static final int AGE_NOT_OK = 11;
+    private static final String PASSWORD_OK = "asdfjkl";
+    private static final String PASSWORD_NOT_OK = "asdf";
+    private static final String LOGIN_OK = "Login";
+    private static final String NAME_OK = "Alina";
+    private static final String NAME_NOT_OK = "Maksym";
+
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(AGE_OK);
+        user.setPassword(PASSWORD_OK);
+        user.setLogin(LOGIN_OK);
+        user.setName(NAME_OK);
+    }
+
+    @Test
+    void user_Not_Ok() {
```

Comment:
```suggestion
    void register_nullUser_notOk() {
```
Name all methods according to the task
![image](https://user-images.githubusercontent.com/20326858/205263573-d0df0fa8-e365-4c6b-8146-39c8869f637e.png)